x-pack/legacy/plugins/lens/public/indexpattern_datasource/field_item.tsx

```diff
@@ -177,6 +177,7 @@ export function FieldItem(props: FieldItemProps) {
       container={document.querySelector<HTMLElement>('.application') || undefined}
       button={
         <DragDrop
+          label={field.name}
```

Comment:
What is the best possible label here?

Reply:
Sorry @wylieconlon I just noticed your comments right after I merged 🤦‍♀ if needed I address your comments in separate PR.

What do you mean 'the best possible'? 🤔